Resources/Storage/SRM2Storage.py

```diff
@@ -82,7 +82,7 @@ def __init__( self, storageName, protocol, path, host, port, spaceToken, wspath
     self.gfalRetry = gConfig.getValue( "/Resources/StorageElements/GFAL_Retry", 3 )
 
     # # set checksum type, by default this is 0 (GFAL_CKSM_NONE)
-    self.checksumType = gConfig.getValue( "/Resources/StorageElements/ChecksumType", 0 )
+    self.checksumType = gConfig.getValue( "/Resources/StorageElements/ChecksumType", None )
```

Comment:
Well, it is more (worse?) than that!
If the default type cannot be cast form what is returned the default value is returned, therefore 
> > -    self.checksumType = gConfig.getValue( "/Resources/StorageElements/ChecksumType", 0 )
always returns 0!!
None is safe, and ‘’ would also be OK
Le 5 mars 2015 à 14:10, Stephane Poss <notifications@github.com> a écrit :

> In Resources/Storage/SRM2Storage.py:
> 
> > @@ -82,7 +82,7 @@ def __init__( self, storageName, protocol, path, host, port, spaceToken, wspath
> >      self.gfalRetry = gConfig.getValue( "/Resources/StorageElements/GFAL_Retry", 3 )
> >  
> >      # # set checksum type, by default this is 0 (GFAL_CKSM_NONE)
> > -    self.checksumType = gConfig.getValue( "/Resources/StorageElements/ChecksumType", 0 )
> > +    self.checksumType = gConfig.getValue( "/Resources/StorageElements/ChecksumType", None )
> Why not putting an empty string? I remember that the "returned type" is infered from the default...
> 
> —
> Reply to this email directly or view it on GitHub.
>